src/optimize/dynamic_dll_plugin/dll_config_model.js

```diff
@@ -234,9 +236,11 @@ function unoptimized() {
 }
 
 export function configModel(rawConfig = {}) {
-  if (IS_KIBANA_DISTRIBUTABLE) {
-    return webpackMerge(common(rawConfig), optimized());
+  const config = extendRawConfig(rawConfig);
+
+  if (IS_KIBANA_DISTRIBUTABLE || !config.dllDevMode) {
```

Comment:
Thoughts on using `unoptimized` if `config.dllDevMode` to keep consistent with the base optimizer [here](https://github.com/elastic/kibana/blob/master/src/optimize/base_optimizer.js#L442)?

Reply:
good catch @tylersmalley! Changed